src/components/AssetEventsPanel/stories/helper.tsx

```diff
@@ -0,0 +1,59 @@
+import { CogniteEvent } from '@cognite/sdk';
+import React from 'react';
+import { AssetEventsPanelStyles } from '../../../interfaces';
+import { fakeEvents, sleep } from '../../../mocks';
+import { MockCogniteClient } from '../../../utils/mockSdk';
+import { ClientSDKProvider } from '../../ClientSDKProvider';
+
+class CogniteClient extends MockCogniteClient {
+  events: any = {
+    list: () => ({
+      autoPagingToArray: () => {
+        return new Promise(async resolve => {
+          await sleep(1000);
+          resolve(fakeEvents);
+        });
```

Comment:
```
      autoPagingToArray: async () => {
          await sleep(1000);
          return fakeEvents
       }
```
I think this could be simplified